src/api/ratelimits.mdx

```diff
@@ -26,9 +26,9 @@ Every API request response includes the following headers:
 - `X-Sentry-Rate-Limit-Reset`
   - The time when the next rate limit window begins and the count resets, measured in UTC seconds from epoch
 - `X-Sentry-Rate-Limit-ConcurrentLimit`
-  - The maximum number of concurrent requests allowed within the window
+  - The maximum number of concurrent ("in flight") requests allowed at any given time
 - `X-Sentry-Rate-Limit-ConcurrentRemaining`
-  - The number of concurrent requests this caller has left on this endpoint within the current window
+  - The number of concurrent requests this caller has left
```

Comment:
this makes it sound like there's a finite pool here that is being filled.

`within the current window` makes it more clear that this is a queue.

Reply:
There is indeed a finite pool being filled. here is the implementation, let me know if you have any questions:

https://github.com/getsentry/sentry/blob/master/src/sentry/scripts/ratelimits/api_limiter.lua

Here is the design document:

https://www.notion.so/sentry/Concurrent-Rate-Limiter-ebd5b7562aed4d01b102d83774a06b11

Reply:
Any updates on where we want to go with this particular change?

Reply:
Ah sure, there's a finite amount of concurrent requests, but its still a queue right? so it will replenish?
Maybe we could tack on "caller has left in the queue"?